config/config.go

```diff
@@ -47,13 +47,17 @@ type Fetcher struct {
 }
 
 type Config struct {
-	AWSConfig  aws.ConfigAWS           `config:",inline"`
-	Fetchers   []*config.C             `config:"fetchers"`
-	KubeConfig string                  `config:"kube_config"`
-	Period     time.Duration           `config:"period"`
-	Processors processors.PluginConfig `config:"processors"`
-	BundlePath string                  `config:"bundle_path"`
-	Benchmark  string                  `config:"config.v1.benchmark"`
+	BenchmarkConfig BenchmarkConfig         `config:"config.v1"`
+	Fetchers        []*config.C             `config:"fetchers"`
+	KubeConfig      string                  `config:"kube_config"`
+	Period          time.Duration           `config:"period"`
+	Processors      processors.PluginConfig `config:"processors"`
+	BundlePath      string                  `config:"bundle_path"`
+}
+
+type BenchmarkConfig struct {
+	ID        string        `config:"benchmark"`
+	AWSConfig aws.ConfigAWS `config:"aws.credentials"`
```

Comment:
I don't see why `AWSConfig` should be a part of `BenchmarkConfig`.
For example, vulnerability management is not a benchmark but we will want to reuse the AWS authentication logic there.

Reply:
I see.
We can keep this structure and rename `BenchmarkConfig` to something more general to include vulnerability management.

Reply:
Besides the integration already sending the AWS credentials as part of the benchmark config, is there a real reason to enforce that coupling? In the future we will have also `GCPConfig` and `AzureConfig` are they all going to be part of an object that also contains the benchmark details?
I think we should have a `CloudConfig` object, that for now holds only aws credentials, while benchmark config should be a separate object.